packages/truffle-debugger/lib/session/index.js

```diff
@@ -42,10 +42,10 @@ export default class Session {
   ready() {
     return new Promise( (accept, reject) => {
       this._store.subscribe( () => {
-        if (this.state.session == "ACTIVE") {
+        if (this.state.session.status == "ACTIVE") {
           accept()
-        } else if (typeof this.state.session == "object") {
-          reject(this.state.session.error);
+        } else if (typeof this.state.session.status == "object") {
```

Comment:
I would advise against making a state variable polymorphic, but in this case I can see why you did it.

Reply:
I didn't do that -- that's preexisting!  This line is only marked as changed because it's now under `session.status` rather than just `session`.

Reply:
It's so much easier to criticize than it is to implement -___-